### PR TITLE
Zib-Patient

### PR DESCRIPTION
**Pull request by AmyKoo1**

### CODE REVIEW

File: zib/StructureDefinitions/zib-Patient.xml [hunk 1]
```diff
@@ -0,0 +1,330 @@
...
+      <mapping>
+        <identity value="zib-patient-v4.3-2024EN" />
+        <map value="NL-CM:0.1.32" />
+        <comment value="DeathIndicator" />
+      </mapping>
+    </element>
+    <element id="Patient.address">
+      <path value="Patient.address" />
+      <definition value="Patient's address information." />
+      <type>
```
Comment (ArdonToonstra) [inline zib/StructureDefinitions/zib-Patient.xml:274]: Hier zou denk ik eigenlijk wel de alias gezet kunnen worden van NL-CM:0.1.4 --> zoals we ook doen voor ContactInformation bijv. Dan kan dat stukje ook uit de known-issues.yml

File: zib/vocabulary/manual/ConceptMap-GeslachtCodelijst-to-AdministrativeGender.xml [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+        <code value="female"/>
+        <equivalence value="equal"/>
+      </target>
+    </element>
+    <element>
+      <code value="UN"/>
+      <target>
+        <code value="other"/>
+        <equivalence value="narrower"/>
+        <comment value="Undifferentiated: the gender of a person could not be uniquely defined as male or female, such as hermaphrodite. Other: neither male nor female and not unknown."/>
```
Comment (ArdonToonstra) [inline zib/vocabulary/manual/ConceptMap-GeslachtCodelijst-to-AdministrativeGender.xml:50]: Zouden we hier ZIB-1239 moeten verwerken?

File: release-notes.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+## zib-Patient
+* `.extension:genderIdentity.value[x]` ; extensions ; Added GenderIdentity on Patient; [ZIB-1693]
+* `.gender` ; terminology ; Added guidance for mapping of 'X' and 'V' in GeslachtCodelijst-to-AdministrativeGender ConceptMap ; [ZIB-1239]
```
Comment (ArdonToonstra) [inline release-notes.md:3]: I don't see this?

**Review by ArdonToonstra — CHANGES_REQUESTED** [review]

File: release-notes.md [hunk 4]
```diff
@@ -0,0 +1,7 @@
...
+* `.multipleBirth[x]:multipleBirthInteger` ; mapping ; Added mapping of MultipleBirthSequence ; [ZIB-1670]
+* `.multipleBirth[x]:multipleBirthInteger` ; textual ; Added guidance: When exporting the data, if `.multipleBirth[x]:multipleBirthInteger` (MultipleBirthSequence) is present and has a value, MultipleBirthIndicator may be set to 'true', since MultipleBirthIndicator and MultipleBirthSequence cannot both be represented at the same time.
```
Comment (ArdonToonstra) [inline release-notes.md]: I did not see the guidance. The definition contains guidance, but is similar to what is defined by the zib. We did not add anything additional. So I removed this.

File: zib/vocabulary/manual/ConceptMap-GeslachtCodelijst-to-AdministrativeGender.xml (hunk 2 above)
Comment (AmyKoo1) [reply zib/vocabulary/manual/ConceptMap-GeslachtCodelijst-to-AdministrativeGender.xml:50]: De omschrijvingen worden niet meegenomen in de conceptmap, alleen de code en die is niet veranderd.

File: release-notes.md [hunk 5]
```diff
@@ -0,0 +1,4 @@
+## Patient (zib-Patient)
```
Comment (ArdonToonstra) [inline release-notes.md:1]: I believe the release notes miss a line for ZIB-2218 for MaritalStatus.

Perhaps add it like 
## MaritalStatus (zib-Patient, nl-core-Patient)
* xxx ; xxxx ; xxx

Comment (ArdonToonstra) [reply release-notes.md:1]: En miss voor Nationality dan:

## Nationality (zib-Patient, nl-core-Patient)
* No significant changes.

Comment (ArdonToonstra) [reply release-notes.md:1]: Missing release note for LanguageProficiency -  ZIB-1877:

File: zib/StructureDefinitions/zib-Patient.xml (hunk 1 above)
Comment (LuudSlagter) [reply zib/StructureDefinitions/zib-Patient.xml]: Zodra zib-Patient en zib-AddressInformation in dezelfde branch zitten, worden de `short` en `alias` weggegooid uit de differential, en dan heb je waarschijnlijk wel weer een known issue nodig.

**Comment by LuudSlagter** [comment]
Only thing that needs to be done before merging is manually correcting some ValueSets (i.e. removing OTH) and refreshing the terminology afterwards.

**Comment by LuudSlagter** [comment]
Note that https://nictiz.atlassian.net/browse/ZIBFHIR2024-381 is an open issue, but this will be solved only after a conclusion has been drawn from the _OTH_ discussion.

**Review by ArdonToonstra — COMMENTED** [review]
Reviewed. Changes look good to me. Also Ada2FHIR part. Some display warninging that not 100% match with FHIR codesytem like for AddressInformation but we will leave it as this requires another change to the zibs for which the previous ones about displays were not really honored as well..

So good to go, but I suggest we firstly try to finish HealthcareProvider, then HealthProfessional and than this one. Then we we will have them in the right order and a lot of warnings will be gone for Patient too.

**Comment by LuudSlagter** [comment]
Make sure to check whether the reference to nl-core-Patient in the nl-core-ContactPerson resolves, as the latter has been merged to main before the former.

**Pull request merged**